docs/user-guide/02_physics/06_npa.md

```diff
@@ -67,10 +67,12 @@ An example of the calculated NPA flux for the two different methods are shown be
 
 # Relevant Namelist Settings
 * `n_npa`: Number of Monte Carlo particles used in MC-NPA calculation
+* `n_pnpa`: Number of Monte Carlo particles used in MC-pNPA calculation
 * `calc_npa`: Calculate NPA flux using the Monte Carlo Method
 * `calc_npa_wght`: Calculate NPA weight function and flux using the weight function method
 * `ne_wght`: Number of energies in weight function calculation
 * `np_wght`: Number of pitches in weight function calculation
+* `nphi_wght`: Number of Gyro-angles in weight function calculation
```

Comment:
did you change the weight function calculations?

Reply:
No.